lib/widgets/message_list.dart

```diff
@@ -892,6 +889,7 @@ class MessageWithPossibleSender extends StatelessWidget {
 
   final MessageListMessageItem item;
 
+  // TODO(#95) unchanged in dark theme?
```

Comment:
Let's squash these three:

14c3f52d0 msglist [nfc]: Comment on _starColor about dark theme
a0ebda6bb msglist [nfc]: Comment on _kMessageTimestampColor about dark theme
61be55b75 msglist [nfc]: Comment with TODO(#95) on mark-as-read button

oh and this previous commit:
53440c7c9 msglist [nfc]: Add TODO on date-separator line about dark theme

They're one line each, all adding comments on the same theme, and none of them call for significant discussion in the commit message.